### ISSUE TITLE
Warp System Through Electrical Networks

### ISSUE THREAD

**Issue by Cooldude2606** [explosivegaming/scenario#188]
**Feature overview**
Name: Warp System Through Electrical Networks
TL;DR: Warps can only travel to warps which are on the same electrical, this will stop long distance travel without proper infrastructure.

**Details**
Every warp has an energy pole, they have for a long time now, and it has long been an idea to make them require energy or fuel in some way. Directly requiring energy would not be possible in a softmod but checking network concetivity would be. This way we could have warps only allow transport to warps on the same electrical network.

**Possible methods**
Each electrical network has its own network id which we can compere to see if two warps are connected to each other. The network id can be stored along side the rest of the warp data and updated when a player walks onto a warp point.

**Comment by Cooldude2606** [comment]
Pull request #190 implements this request and has been merged into dev. Clossing issue.
